Javascript errors with 7.0.0.alpha9

I hit this issue today when working through the tutorial notebooks with some students, who noticed that styles (like button color) were being ignored.

This is in a clean environment with ipywidgets 7.0.0.alpha9, notebook 5.0.0

I think there are two issues, I think. 

One happens when the notebook is loaded, prior to running any code:

```
utils.js:62 Failed to load extension: ["nbextensions/jupyter-widgets/extension"] Error: Script error for "nbextensions/jupyter-widgets/extension"
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js?v=6da8be3…:165)
    at HTMLScriptElement.onScriptError (require.js?v=6da8be3…:1732)
(anonymous) @ utils.js:62
require.js?v=6da8be3…:1952 GET http://localhost:8888/nbextensions/ipyvolume/extension.js?v=20170706205300 
req.load @ require.js?v=6da8be3…:1952
load @ require.js?v=6da8be3…:1679
load @ require.js?v=6da8be3…:829
fetch @ require.js?v=6da8be3…:819
check @ require.js?v=6da8be3…:851
enable @ require.js?v=6da8be3…:1177
enable @ require.js?v=6da8be3…:1550
(anonymous) @ require.js?v=6da8be3…:1162
(anonymous) @ require.js?v=6da8be3…:131
each @ require.js?v=6da8be3…:56
enable @ require.js?v=6da8be3…:1114
init @ require.js?v=6da8be3…:783
(anonymous) @ require.js?v=6da8be3…:1453
```

Looking in the environment, the contents of `share/jupyter/nbextensions` is `bqplot` and `jupyter-js-widgets`, but no `jupyter-widgets`.

Likely because of that (I'm guessing), when you execute some simply code like

```python
import ipywidgets as widgets
print(widgets.__version__)
b = widgets.Button(description='Click me')
b
```
more javascript errors are thrown:

```
utils.js:84 Error: Could not create child view
    at promiseRejection (utils.js:84)
    at <anonymous>
promiseRejection @ utils.js:84
utils.js:84 Error: Could not add styleView to DOMWidgetView
    at promiseRejection (utils.js:84)
    at <anonymous>
promiseRejection @ utils.js:84
manager-base.js:87 Uncaught (in promise) TypeError: ViewType is not a constructor
    at manager-base.js:87
    at <anonymous>
(anonymous) @ manager-base.js:87
manager-base.js:87 Uncaught (in promise) TypeError: ViewType is not a constructor
    at manager-base.js:87
    at <anonymous>
(anonymous) @ manager-base.js:87
```

--- a/ipywidgets/widgets/widget_layout.py
+++ b/ipywidgets/widgets/widget_layout.py
@@ -24,8 +24,8 @@ class Layout(Widget):
     """
 
     _view_name = Unicode('LayoutView').tag(sync=True)
-    _view_module = Unicode('@jupyter-widgets/base')
-    _view_module_version = Unicode('3.0.0')
+    _view_module = Unicode('@jupyter-widgets/base').tag(sync=True)
+    _view_module_version = Unicode('3.0.0').tag(sync=True)
     _model_name = Unicode('LayoutModel').tag(sync=True)
 
     # Keys

--- a/ipywidgets/widgets/widget_style.py
+++ b/ipywidgets/widgets/widget_style.py
@@ -10,5 +10,7 @@ from .widget import Widget
 class Style(Widget):
     """Style specification"""
 
-    _view_name = Unicode('StyleView').tag(sync=True)
     _model_name = Unicode('StyleModel').tag(sync=True)
+    _view_name = Unicode('StyleView').tag(sync=True)
+    _view_module = Unicode('@jupyter-widgets/base').tag(sync=True)
+    _view_module_version = Unicode('3.0.0').tag(sync=True)

--- a/packages/controls/src/widget_description.ts
+++ b/packages/controls/src/widget_description.ts
@@ -14,7 +14,6 @@ class DescriptionStyleModel extends StyleModel {
     defaults() {
         return {...super.defaults(),
             _model_name: 'DescriptionStyleModel',
-            _view_module: '@jupyter-widgets/controls',
             _model_module: '@jupyter-widgets/controls',
         };
     }

